exercises/ex05-offchain-workers/transaction-function/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+#![cfg_attr(not(feature = "std"),no_std)]
```

Comment:
I don't understand what this new section does.
It doesn't add much to what has already been demonstrated in the previous section